src/ng1/viewsBuilder.ts

```diff
@@ -0,0 +1,107 @@
+/** @module ng1 */ /** */
+import {State} from "../state/stateObject";
+import {pick, forEach} from "../common/common";
+import {ViewConfig, ViewContext} from "../view/interface";
+import {Ng1ViewDeclaration} from "./interface";
+import {ViewService} from "../view/view";
+import {isInjectable} from "../common/predicates";
+import {services} from "../common/coreservices";
+import {trace} from "../common/trace";
+import {Node} from "../path/node";
+import {TemplateFactory} from "../view/templateFactory";
+import {ResolveContext} from "../resolve/resolveContext";
+
+export const ng1ViewConfigFactory = (node, view) => new Ng1ViewConfig(node, view);
+
+/**
+ * This is a [[StateBuilder.builder]] function for angular1 `views`.
+ *
+ * When the [[StateBuilder]] builds a [[State]] object from a raw [[StateDeclaration]], this builder
+ * handles the `views` property with logic specific to angular-ui-router (ng1).
+ *
+ * If no `views: {}` property exists on the [[StateDeclaration]], then it creates the `views` object
+ * and applies the state-level configuration to a view named `$default`.
+ */
```

Comment:
Moved the existing `views` builder into ng1 specific code.  ng1/ng2 require different information.